source/reference/configuration.txt

```diff
@@ -144,3 +154,77 @@ Options
 
    To set the ``verbosity`` setting from the command line, see the
    :option:`--verbosity` option.
+
+.. _c2c-reconfigure-mid-migration: 
+
+Reconfigure ``mongosync`` During Synchronization
+================================================
+
+.. important:: 
+
+   You can't adjust the :setting:`cluster0`, :setting:`cluster1`, or :setting:`id` settings during a migration. 
```

Comment:
Is this list going to need further updates as further settings are added?

Something like this might be more maintainable:
> Only the following settings are adjustable during a migration. Others *MUST* remain unchanged between `mongosync` restarts.